fpmsyncd/routesync.cpp

```diff
@@ -25,34 +25,32 @@ void RouteSync::onMsg(int nlmsg_type, struct nl_object *obj)
 {
     struct rtnl_route *route_obj = (struct rtnl_route *)obj;
     struct nl_addr *dip;
-    char ifname[MAX_ADDR_SIZE + 1] = {0};
-    uint32_t ipv4;
-    int prefix;
+    char destipprefix[MAX_ADDR_SIZE + 1] = {0};
 
     dip = rtnl_route_get_dst(route_obj);
-    /* Supports IPv4 address only for now */
-    if (rtnl_route_get_family(route_obj)  != AF_INET)
+    nl_addr2str(dip, destipprefix, MAX_ADDR_SIZE);
+    SWSS_LOG_DEBUG("destipprefix=%s\n", destipprefix);
```

Comment:
not sure we need this debug log, at minimum it needs more be descriptive, like receive new route message  dest ip prefix %s